sort.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+)
+
+func main() {
```

Comment:
Typically the `main` function would live in a file called `main.go` so the entry point is clear.